alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorV2RestApiIT.kt

```diff
@@ -126,6 +127,74 @@ class SecureMonitorV2RestApiIT : AlertingRestTestCase() {
         ensureNumMonitorV2s(0)
     }
 
+    fun `test update monitor that queries index user doesn't have access to fails`() {
+        if (!isHttps()) {
+            return
+        }
+
+        // RBAC is out of scope for this test, so give all users and requests the same one
+        val backendRole = "backend_role_a"
```

Comment:
[Not blocking] We can disable filter by backend role setting so this is not needed to be added.